scripts/globals/znm_data.lua

```diff
@@ -0,0 +1,285 @@
+-----------------------------------
+---- ZNM Data Tables
+---- Sanraku Trophies, Pop Items, Seals, etc.
+-----------------------------------
+
+xi = xi or {}
+xi.znm = xi.znm or {}
+
+-----------------------------------
+--- General Helper Variables
+-----------------------------------
+
+-- Soultrapper Variables
```

Comment:
I wonder if these could be broken out into the settings

Reply:
Could be. Are Settings the future or are modules?

Reply:
Tweaking numbers: settings
Random custom non-retail bullshit: modules